applications/api-cat/src/test/java/no/acat/restapi/HarvestControllerTest.java

```diff
@@ -0,0 +1,84 @@
+package no.acat.restapi;
+
+import no.acat.harvester.HarvestQueue;
+import no.dcat.shared.testcategories.UnitTest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.stubbing.Answer;
+
+import java.lang.reflect.Method;
+
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@Category(UnitTest.class)
+public class HarvestControllerTest {
+
+  HarvestQueue harvestQueue;
+  @Spy @InjectMocks private HarvestController harvestController;
+
+  @Before
+  public void setup() {
+    MockitoAnnotations.initMocks(this);
+    harvestQueue = mock(HarvestQueue.class);
+  }
+
+  @Test
+  public void add_one_task_to_harvest_quene() {
```

Comment:
CamelCase?

Reply:
brukte SnakeCase, men for å være konsistent med rest av prosjektet endrer jeg til CamelCase :)